mcs/class/System.Core/basic_System.Core.dll.sources

```diff
@@ -1 +1,7 @@
 #include net_4_x_System.Core.dll.sources
+
+System.IO.Pipes/AnonymousPipeServerStream.NotSupported.cs
```

Comment:
Would be nice to have this separated in e.g. pipes_pns.sources and included instead of copied

Reply:
Sure, I can do that.